### PR TITLE
Fix overlapping characters

### PR DESCRIPTION
**Pull request by caleb-mabry**
Explains what it does!

### CODE REVIEW

**Comment by stonedDiscord** [comment]
why does the char have _img but the pair doesn't
this is confusing

**Comment by stonedDiscord** [comment]
you can just import the transparent png instead of redefining it in setEmote.js

**Comment by caleb-mabry** [comment]
Resolved both of @stonedDiscord 's comments

**Comment by stonedDiscord** [comment]
i had the impression this adds lag but it i checked again against master and this makes it slightly better i think

good one

File: webAO/client/setEmote.js [hunk 1]
```diff
@@ -10,22 +11,18 @@ const setEmote = (AO_HOST, client, charactername, emotename, prefix, pair, side)
...
-  const webp_s = document.getElementById(`client_${position}${pairID}_webp`);
-  const extensionsMap = {
-    '.gif': gif_s,
-    '.png': png_s,
-    '.apng': apng_s,
-    '.webp': webp_s,
-  };
-
-  for (const [extension, htmlElement] of Object.entries(extensionsMap)) {
+  for (const extension of extensionsMap) {
```
Comment (oldmud0) [inline webAO/client/setEmote.js:22]: wait can you remind me again why we do this sequentially instead of in parallel

**Pull request merged**